app/assets/stylesheets/dashboards.scss

```diff
@@ -24,3 +24,28 @@
 .bootstrap_toggle_spacing .checkbox {
   margin-top: 0px;
 }
+
+.pt-phone,.pt-LMP {
```

Comment:
Can we collapse these into one class since they're doing the same thing?

Should we also apply a css class to the appointment date and status tds? Interested in your opinion here.